lib/pure/strutils.nim

```diff
@@ -189,6 +189,33 @@ proc isUpper*(s: string): bool {.noSideEffect, procvar,
   for c in s:
     result = c.isUpper() and result
 
+proc isTitle*(s: string): bool {.noSideEffect, procvar,
```

Comment:
IMO this should go into the Unicode module and be Unicode ready only. Who needs an ASCII based isTitle nowadays? ``strutils.toLower`` etc. only working with Ascii is bad enough, but warranted for eg. HTTP protocols.